imagetest/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN cd test_suites; for suite in *; do \
   cd ..; \
   done
 
-FROM alpine:edge
+FROM alpine:3.17
```

Comment:
Given all of the warnings edge has for potential data loss, I definitely agree we shouldn't be using it. 

Pinning to a specific version has proven to be an approach that we struggle with to maintain. 
When I look at https://hub.docker.com/_/alpine I see `3.17` and `latest` point to the same build.

I'd suggest changing this to `FROM alpine:latest`, so we keep up to date on their latest stable release.

Reply:
Sounds good to me. I'll change it quickly.

Reply:
Done